theme_crowdfunding/static/src/style/custom.scss

```diff
@@ -574,151 +583,95 @@ footer {
   }
 }
 
-// Global styles
-.text {
-  &--bold {
-    color: $gray-700;
-    font-weight: 700;
-    font-size: 1.1rem;
+.donation-type {
+  background-color: $blue-compassion;
+
+  &__card-icon {
+    height: 120px;
+    width: 120px;
+    background-color: $blue-compassion;
+    border-radius: 50%;
+    margin: 1rem auto;
   }
 
-  &--faded {
-    color: $gray-600;
-    font-weight: 400;
-    font-size: 0.8rem;
+  &__project-card-image {
+    height: 12rem;
+    object-fit: cover;
   }
 }
 
-// progress bar
-.progress {
-    &-bar {
-        background-color: $blue-compassion !important;
-    }
-}
+.donation-form-amount {
+  background: linear-gradient(180deg, $blue-compassion 90%, $white 90%);
 
-.pagination {
-   justify-content: center;
+  input[type="number"] {
+    width: 80%;
+    border: 2px solid $blue-compassion;
+    border-radius: 4px;
+    padding: 0.5rem 1rem;
+  }
 }
 
-.center {
-  display: block;
-  margin-left: auto;
-  margin-right: auto;
-  width: 50%;
+// Selectable cards
+.card-input-element + .card:hover {
+  cursor: pointer;
 }
 
-*, *:after, *:before {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-  font-family: "Open Sans";
+.card-input-element:checked + .card {
+  background-color: lighten($blue-compassion, 50%) !important;
+  transition: background-color 0.3s;
 }
 
+.fake-btn {
+  background-color: $blue-compassion;
+  color: $white;
+  padding: 0.5rem 1rem;
+}
 
-
-
-$complete: #2ECC71;
-$active: #3498DB;
-$mute: #DFE3E4;
-
-.progress {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-  display: table;
-  table-layout: fixed;
+// Fix cms.form checkboxes
+.field-boolean {
```

Comment:
Won't this change the styles for all forms in all pages/themes ? Maybe it's risky and should use more specific selectors or is it only applied when the crowdfunding_theme is active (would be good)?

Reply:
It was my assumption that it was only applied when the theme is active, otherwise the theme wouldn't be working. From what I tested this is the case.